js/client-battle-tooltips.js

```diff
@@ -409,6 +427,301 @@ var BattleTooltips = (function () {
 		return text;
 	};
 
+	BattleTooltips.prototype.findModifier = function (pokemon, statName) {
+		var modifier = 1;
+		if (pokemon.boosts) {
+			if (pokemon.boosts[statName]) {
+				if (pokemon.boosts[statName] > 0) {
+					var goodBoostTable = [1, 1.5, 2, 2.5, 3, 3.5, 4];
+					//var goodBoostTable = ['Normal', '+1', '+2', '+3', '+4', '+5', '+6'];
+					modifier = goodBoostTable[pokemon.boosts[statName]];
+				} else {
+					var badBoostTable = [1, 0.67, 0.5, 0.4, 0.33, 0.29, 0.25];
+					//var badBoostTable = ['Normal', '&minus;1', '&minus;2', '&minus;3', '&minus;4', '&minus;5', '&minus;6'];
+					modifier =  badBoostTable[-pokemon.boosts[statName]];
+				}
+
+			}
+		}
+
+		//check for burn, paralysis, guts, quick feet
```

Comment:
As an aside, the goodboosttable and badboosttable made in lines 435 through 441 are nearly directly copied from here: https://github.com/Zarel/Pokemon-Showdown-Client/blob/8dc23233fa344256f0d3f6e094498cbed3b8341a/js/battle.js#L608 

The particular comment linked and several others nearby do not have spaces between // and the comment. This is something that should probably be fixed in those places.